lib/ApiClient.php

```diff
@@ -290,10 +305,9 @@ private function getFilePathCache($url): string
      */
     private function getJsonFromCache(string $url): ?string
     {
-        $file_path = $this->getFilePathCache($url);
-        $cache_path = $file_path . "/cache.json";
-        if (file_exists($cache_path) && !$this->isCacheExpired($cache_path)) {
-            return file_get_contents($cache_path);
+        $cache_file_path = $this->getFilePathCacheFromUrl($url);
```

Comment:
This is probably the starting point. The condition isCacheExpired prevented the function to retrieve the cache.